recipe/meta.yaml

```
@@ -1,29 +1,34 @@
 {% set name = "hvac" %}
-{% set version = "0.11.2" %}
+{% set version = "2.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f905c59d32d88d3f67571fe5a8a78de4659e04798ad809de439f667247d13626
+  sha256: 1b85e3320e8642dd82f234db63253cda169a817589e823713dc5fca83119b1e2
 
 build:
   number: 0
-  noarch: python
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed
 
 requirements:
   host:
+    - python
     - pip
-    - python >=3.6
-    - requests >=2.21.0
+    - poetry-core >=1.0.0
   run:
-    - python >=3.6
-    - requests >=2.21.0
+    - python
+    - requests >=2.27.1,<3.0.0
+  # pyhcl is missing
+  # run_constrained:
+  #   - pyhcl >=0.4.4,<0.5.0
```

```suggestion
  run_constrained:
    - pyhcl >=0.4.4,<0.5.0
```
You can uncomment the run_constrained. If pyhcl ever gets added, it will just work :)

---

Done